bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableCoreConstants.java

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.config;
+
+import com.google.api.core.InternalApi;
+import com.google.cloud.bigtable.grpc.async.BulkMutation;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import io.grpc.Status;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+/** Default values for various bigtable operation configuration. */
+@InternalApi("For internal usage only")
+public class BigtableCoreConstants {
+
+  /** For internal use only - public for technical reasons. */
+  @InternalApi("For internal usage only")
+  public static final String BIGTABLE_EMULATOR_HOST_ENV_VAR = "BIGTABLE_EMULATOR_HOST";
+
+  /** For internal use only - public for technical reasons. */
+  @InternalApi("For internal usage only")
+  public static final String BIGTABLE_ADMIN_HOST_DEFAULT = "bigtableadmin.googleapis.com";
+
+  /** For internal use only - public for technical reasons. */
+  @InternalApi("For internal usage only")
+  public static final String BIGTABLE_DATA_HOST_DEFAULT = "bigtable.googleapis.com";
+
+  /** For internal use only - public for technical reasons. */
+  @InternalApi("For internal usage only")
+  public static final String BIGTABLE_BATCH_DATA_HOST_DEFAULT = "batch-bigtable.googleapis.com";
+
+  /** For internal use only - public for technical reasons. */
+  @InternalApi("For internal usage only")
+  public static final int BIGTABLE_PORT_DEFAULT = 443;
+
+  /** For internal use only - public for technical reasons. */
+  @InternalApi("For internal usage only")
+  public static final int BIGTABLE_DATA_CHANNEL_COUNT_DEFAULT = getDefaultDataChannelCount();
+
+  /**
+   * Constant <code>BIGTABLE_APP_PROFILE_DEFAULT=""</code>, defaults to the server default app
+   * profile
+   *
+   * <p>For internal use only - public for technical reasons.
+   */
+  @InternalApi("For internal usage only")
+  public static final String BIGTABLE_APP_PROFILE_DEFAULT = "";
+
+  /** @deprecated This field will be removed in future versions. */
+  @Deprecated public static final String BIGTABLE_CLIENT_ADAPTER = "BIGTABLE_CLIENT_ADAPTER";
+
```

Comment:
Would it make sense to remove this deprecated constant now?

Reply:
Thanks for pointing it out, will remove before going public.